crates/compiler/builtins/bitcode/src/str.zig

```diff
@@ -90,13 +90,13 @@ pub const RocStr = extern struct {
     }
 
     pub fn deinit(self: RocStr) void {
-        if (!self.isSmallStr()) {
-            utils.decref(self.str_bytes, self.str_len, RocStr.alignment);
-        }
+        self.decref();
     }
 
     fn decref(self: RocStr) void {
-        self.deinit();
+        if (!self.isSmallStr()) {
+            utils.decref(self.str_bytes, self.str_capacity, RocStr.alignment);
+        }
```

Comment:
swapped bodies here. `decref` is nicer semantically to me. `deinit` still feels like `free` somehow, which is not quite what we actually do here.

Reply:
also, the core change is here: `self.str_len` is replaced by `self.str_capacity`